prac_06/guitar_test.py

```diff
@@ -0,0 +1,15 @@
+"""
+CP1404/CP5632 Practical - Guitar Test
+"""
+from guitar import Guitar
+
+guitar = Guitar("Gibson L-5 CES",1922,16035.40)
+another_guitar = Guitar('Another Guitar', 2013)
+first_line = input(f'{guitar.name} get_age() - Expected ' )
```

Comment:
Did you consider putting the f string into the print statement below instead of using input?